Update AppGui.java

miftahulihsan opened the pull request.

miftahulihsan committed:
Update App.java

Evan-aja commented on AppSource/App.java:
```diff
@@ -6,11 +6,11 @@
 public class App {
     Scanner input = new Scanner(System.in);
     ArrayList<Makanan> makananArrayList;
-    ArrayList<Minum> minumanArrayList;
```
San, untuk nama kelas nya bisa disamain sama gui? Soalnya di gui buat window nya ada yang namanya sama kayak source, dan kalau mau bikin kode sekalian di AppGui nya ya, yang disini gunanya buat dipindah ke gui, nanti dihapus package ini

Evan-aja commented:
> 

Jangan lupa kasih penjelasan disini juga, biar tau yang harus dicari yang mana

miftahulihsan commented:
Yang diubah:
-ditambah atribut jenisMenu di class abstrak Menu
-ditambah method getJenisMenu di class Makanan dan Minuman
-ditambah syarat ditambahnya makanan/minuman ke dalam list di dalam method panelTambahMenu

miftahulihsan committed:
Update App.java

Class Makanan diganti Makan

miftahulihsan committed:
Update AppGui.java

Update class App, penambahan setJenisMenu, dan if condition untuk tambah makanan yang sudah ada di dalam list

miftahulihsan committed:
Update App.java

miftahulihsan commented:
itu udah aku ubah ya vann

Evan-aja reviewed (changes requested):
San, buat yang maksudnya dipindah ke gui itu bukan cli nya, maksudnya di implementasikan ke sini, jadi bukan full copy

miftahulihsan committed:
Update AppGui.java

Perubahan di class AppGui.java mulai dari baris 192 - 209. Lalu di psvm ditambahkan IOException

miftahulihsan committed:
Update Makanan.java

update punya evan sebelumnya

miftahulihsan committed:
Update Minuman.java

update punya evan sebelumnya

miftahulihsan committed:
Update ModeKasir.java

update punya evan sebelumnya

miftahulihsan committed:
Update StrukTotal.java

update punya evan sebelumnya

miftahulihsan committed:
Update Tambah.java

update punya evan sebelumnya

Evan-aja commented:
broken
